.github/workflows/ci.yml

```diff
@@ -25,3 +25,7 @@ jobs:
       - run: yarn install --frozen-lockfile
       - run: yarn prettier --check .
       - run: yarn run lint
+      - name: Run Cypress tests
+        uses: cypress-io/github-action@v4
+        with:
+          start: yarn dev
```

Comment:
I reverted this by force pushing. I suggest we do this later, because I also need to create a full-scale ACMS project in the CI environment before I can run Cypress tests anyway. That'll be its own PR.